rust/core-lib/src/rpc.rs

```diff
@@ -281,6 +284,15 @@ impl<'a> EditCommand<'a> {
             "redo" => Ok(Redo),
             "cut" => Ok(Cut),
             "copy" => Ok(Copy),
+
+            "find" => params.as_object().map(|dict| {
+                let chars = dict_get_string(dict, "chars");
+                let case_sensitive = dict_get_bool(dict, "case_sensitive").unwrap_or(false);
```

Comment:
Unfortunately, this will panic with `panicked at 'no entry found for key'` if `case_sensitive` is not provided at all. Using `#[derive(Deserialize)]` would probably be a nice solution, but maybe better suited for a PR that converts all RPC methods to `#[derive(Deserialize)]`?